### PR TITLE
Feat: add website header

### PR DESCRIPTION
**Pull request by vutuanlinh2k2**
## Summary of changes
_Provide a detailed description of proposed changes._
- Adding the header for the Tangle Website (Desktop, Tablet and Mobile)

### Proposed area of change
_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [ ] `apps/stats-dapp`
- [ ] `apps/webbsite`
- [x] `apps/tangle-website`
- [ ] `libs/webb-ui-components`

### Reference issue to close (if applicable)
_Specify any issues that can be closed from these changes (e.g. Closes #233)._
- Closes 

### Screen Recording
_If possible provide a screen recording of proposed change._

https://user-images.githubusercontent.com/69841784/227409616-83b8c6a9-2321-45ab-88ad-dad552a1a5cb.mov

-----
### Code Checklist 
_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by devpavan04 — DISMISSED** [review]
@vutuanlinh2k2 I would suggest moving `TangleLogo` into `webb-ui-components` but LGTM 👍🏼

File: apps/tangle-website/src/components/TangleLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  size?: 'sm' | 'md' | 'lg';
+}
+
+const defaultLogoSize = {
+  width: 114,
+  height: 50,
+  viewBox: '0 0 114 50',
+};
+
+export const TangleLogo: React.FC<LogoProps> = (props) => {
```
[the remark this answers is not in the document]
Comment (vutuanlinh2k2) [reply apps/tangle-website/src/components/TangleLogo.tsx]: @devpavan04 I have pushed the the modify code that move TangleLogo component to the webb-ui-components as you suggested!

**Review by AtelyPham — CHANGES_REQUESTED** [review]
Please consider removing copying files, fonts @vutuanlinh2k2. Taking a look on the [`develop`](https://github.com/webb-tools/webb-dapp/tree/develop/apps/tangle-website) branch, we should only add necessary files for the tangle.

**Review by AtelyPham — COMMENTED** [review]
Files and directories to remove:

- `public/fonts` (consider moving supported fonts into `webb-ui-kit`)
- `public/static/assets/community-texture.png`
- `constants/index.ts` (consider moving the `constants` to the `dapp-config` lib for sharing)
- `libs/notion` (we don't need this at this time)

For the headings and subheadings, if you want to share between `webbsite` app and `tangle-website`, please consider moving them into the `webb-ui-kit`. @vutuanlinh2k2

**Review by AtelyPham — COMMENTED** [review]
Great job on this PR! Just one quick thing - could you please switch out all the fonts here with the default one and make sure to remove them from the tailwind.config.js and global.css files too? We can always import the fonts in a later PR. Thank you.

**Review by AtelyPham — APPROVED** [review]
LGTM 👍

**Pull request merged**